velox/core/QueryConfig.h

```diff
@@ -354,6 +354,15 @@ class QueryConfig {
   static constexpr const char* kDriverCpuTimeSliceLimitMs =
       "driver_cpu_time_slice_limit_ms";
 
+  /// The max bytes number of normalized keys for each row in the prefixsort.
+  /// The default value is max uint_32, and set to 0 to disable prefixsort.
+  static constexpr const char* kPrefixSortMaxNormalizedKeySize =
+      "prefixsort_max_normalized_key_size";
+
+  /// The minimum number of rows that can trigger the prefixsort. This is set to
+  /// '130' by default.
+  static constexpr const char* kPrefixSortThreshold = "prefixsort_threshold";
```

Comment:
@skadilover : How do you come up with this ? I think this would depend on the machine capabilities. Would you give us a special script to tune this on different machines ?

Reply:
velox/exec/benchmarks/
PrefixSortBenchmark.cpp

@aditi-pandit